wiki/Beatmap_Editor_Guides/Music_Theory/en.md

```diff
@@ -8,7 +8,7 @@ Osu! is primarily a music game, and beatmaps are usually created with the idea t
 Part 1: Breakdown of common time and techniques
 -----------------------------------------------
 
-![](Musictheory1.png‎)
+![](MusicTheory1.png)
```

Comment:
L11: Put a text inside ``[]`` to indicate which should be there if the image fails to load.